server/test/integration/dashboardTest/submitEditClientTest.js

```diff
@@ -670,6 +700,7 @@ describe('dashboard.js integration tests: submitEditClient', () => {
         isDisplayed: '',
         isSendingAlerts: '',
         isSendingVitals: '',
+        language: '',
```

Comment:
Because the `language` is now a required field, I would have expected the error log to include `location (Invalid value)`, but this doesn't seem to have changed. Could you please check that you are making `location` a required field in all the places that it needs to be for both this form and for the `submitNewClient` as well?

Reply:
Same thing for "describe('for an empty request'" test below. I was expecting to see `location (Invalid value)` in its error message.